src/frontend/src/lib/stores/token-group.store.ts

```diff
@@ -0,0 +1,11 @@
+import { initStorageStore } from '$lib/stores/storage.store';
+
+export interface TokenGroupData {
+	expanded: boolean;
+}
+
+export interface TokenGroupsData {
+	[key: string]: TokenGroupData;
+}
+
+export const tokenGroupStore = initStorageStore<TokenGroupsData>({ key: 'token-groups' });
```

Comment:
if you plan for the keys to be TokenId (I imagine it will be the native token ID of the group), you can use directly

`export const tokenGroupStore = initCertifiedSetterStore<TokenGroupData>({ key: 'token-groups' });`

defaulting the certified part to either `true` or `false`, you decide

Basically, this store will have the structure you need `[token] : data` and some useful `set` and `reset` functions

Reply:
sounds good, need to refactore the store to deal with generic key / symbol. Currently only tokenId is supported.

Reply:
`symbol` can be persisted in the local storage?

Reply:
No, symbol cannot be persisted in local storage. The CertifiedSetterStore seems not to store to local storage and be implemented on TokenId (symbol).